content/pt-br/docs/tasks/configure-pod-container/migrate-from-psp.md

```diff
@@ -0,0 +1,389 @@
+1. Revise as permissões do namespace
+2. Simplifique e padronize as `PodSecurityPolicies`
+3. Atualize os namespaces
+   1. Identifique um nível apropriado de segurança de Pod
+   2. Verifique o nível de segurança do Pod
+   3. Aplique o nível de segurança do Pod
+   4. Desvie o `PodSecurityPolicy`
+4. Revise os processos de criação do namespace
+5. Desabilite o `PodSecurityPolicy`
+
+## 0. Decida se a Segurança de Admissão de Pod é o correto para você {#is-psa-right-for-you}
+
+Segurança de Admissão de Pod foi projetado para atender às necessidades de segurança mais 
+comuns, e para fornecer um conjunto padrão de níveis de segurança entre clusters. 
+No entanto, é menos flexível do que o `PodSecurityPolicy`. Notavelmente, os seguintes 
+recursos são suportados pela `PodSecurityPolicy` mas não pela Segurança de Admissão do Pod:
+
+- **Definindo restrições de segurança padrão** - Segurança de Admissão de Pod 
+  é um controlador de admissão não mutável, significando que não modificará 
+  os pods antes de validá-los. Se você estava confiando nesse aspecto do PSP, 
+  você precisará modificar suas cargas de trabalho para atender às restrições 
+  de segurança do Pod, ou usar um [Webhook de admissão mutante]
+  (/docs/reference/access-authn-authz/extensible-admission-controllers/)
+  para fazer essas mudanças. Veja [Simplificando e padronizando `PodSecurityPolicies`]
+  (#simplify-psps) abaixo, para mais detalhes.
+
+- **Controle de granularidade sobre a definição de política** 
+- A Segurança de Admissão de Pod somente suporta 
+  [3 níveis padrão](/docs/concepts/security/pod-security-standards/).
+  Se você precisar de mais controle sobre restrições específicas, então você 
+  precisará usar um [Webhook de validação de admissão]
+  (/docs/reference/access-authn-authz/extensible-admission-controllers/) 
+  para fazer cumprir essas políticas.
+
+- **Granularidade da política de sub-namespace** - `PodSecurityPolicy` permite 
+  vincular políticas diferentes a contas de serviço ou usuários, mesmo dentro 
+  de um único namespace. Esta abordagem tem muitas armadilhas e não é recomendada, 
+  mas se você precisa desse recurso de qualquer maneira, você vai precisar usar 
+  um webhook de terceiros. A exceção a isso é se você apenas precisar liberar 
+  completamente usuários específicos ou [Classes de Runtime](/docs/concepts/containers/runtime-class/),
+  nesse caso a Segurança de Admissão do Pod irá expor alguma
+  [Configuração estática para isenções](/docs/concepts/security/pod-security-admission/#exemptions).
+
+Mesmo que a Segurança de Admissão de Pod não atenda a todas as suas necessidades, 
+ela foi projetada para ser _complementar_ a outros mecanismos de aplicação de políticas, 
+e pode fornecer um controle de falha útil, rodando ao lado de outro Webhook de admissão.
+
+
+## 1. Revise as permissões de namespace {#review-namespace-permissions}
+
+A Admissão de segurança do Pod é controlada através de [rótulos em namespaces]
```

Comment:
Following your standard capital letters "Admissão de Segurança do Pod" 

This link "rótulos em namespaces" is broken. Remove the newline after the brackspaces.